docs/admin/Offline_Cert_Renewal.md

```diff
@@ -0,0 +1,178 @@
+Offline Cert Renewal
+====================
+
+## Overview
+
+PKI server provides a mechanism to recover from expired system certificates. This mechanism can also be
+used to renew the certificates before they expire. There are 2 ways to renew the certs
+
+1. [Automated Renewal Process](#Automated-Renewal-Process)
+2. [Manual Renewal Process](#Manual-Renewal-Process) 
+
+It is assumed that you have the following:
+* Valid CA signing cert
+* Valid admin cert
+
+To verify these assumptions are valid:
+
+1. List details of all system certificates. (Note down the `<cert ID>` of the certs that needs to be renewed)
+
+    ````
+    # pki-server cert-find
+    ````
+
+2. Check details of admin cert
+
+    ````
+    # certutil -L \
+    -d <client NSS DB dir> \
+    -n <admin cert nickname>
+    ````
+
+3. Check status of PKI server
+
+    ````
+    # systemctl status pki-tomcatd@pki-tomcat
+    ````
+
+## Automated Renewal Process
+
+One line tool that fixes all certificates:
+
+    # pki-server cert-fix \
+    -n <admin nickname> \
+    -d <NSS db path> \
+    -c <NSS client DB password>
+
+
+One line tool to fix one particular certificate:
+
+    # pki-server cert-fix --cert <cert ID> \
+    -n <admin nickname> \
+    -d <NSS db path> \
+    -c <NSS client DB password>
+
+For all available options, you can type:
+
+    $ pki-server cert-fix --help
+
+## Manual Renewal Process
+### Initialization
+
+It is recommended to run the following steps to ensure that `CS.cfg` and NSS db are synchronized and
+that the server can operate without any issues.
+
+1. Disable self tests. Remove the following line from CS.cfg for the <subsystem> you are renewing.
```

Comment:
Remove the instruction to manually edit the CS.cfg. It's no longer necessary since we already have the pki-server selftest-disable command.